docs/Server.md

```diff
@@ -631,18 +639,21 @@ fastify.setReplySerializer(function (payload, statusCode){
 })
 ```
 
-<a name="set-schema-compiler"></a>
-#### setSchemaCompiler
-Set the schema compiler for all routes [here](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#schema-compiler).
+<a name="set-validator-compiler"></a>
+#### setValidatorCompiler
+Set the schema validator compiler for all routes [here](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#schema-validator).
 
-<a name="set-schema-resolver"></a>
-#### setSchemaResolver
-Set the schema `$ref` resolver for all routes [here](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#schema-resolver).
+<a name="set-serializer-resolver"></a>
+#### setSerializerCompiler
+Set the schema serializer compiler for all routes [here](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#schema-serializer).
 
+<a name="validator-compiler"></a>
+#### validatorCompiler
+This property can be used to get the schema validator. If not set, it will be `null` until the server starts.
 
-<a name="schema-compiler"></a>
-#### schemaCompiler
-This property can be used to set the schema compiler, it is a shortcut for the `setSchemaCompiler` method, and get the schema compiler back for all routes.
+<a name="serializer-compiler"></a>
+#### serializerCompiler
+This property can be used to get the schema serializer. If not set, it will be `null` until the server starts.
```

Comment:
What will it be after the server starts?

Reply:
As above: I will add this info 👍